gc/base/ParallelDispatcher.cpp

```diff
@@ -649,12 +649,6 @@ MM_ParallelDispatcher::reinitAfterFork(MM_EnvironmentBase *env, uintptr_t newThr
 }
 
 #if defined(J9VM_OPT_CRIU_SUPPORT)
-void
-MM_ParallelDispatcher::contractThreadPool(MM_EnvironmentBase *env, uintptr_t newThreadCount)
```

Comment:
Perhpas we can leave contract/expandThreadPool methods as private and let them do the real work and have prepareForCheckpoint/Restore as public wrappers to call contract/expand.

We gain a bit on self-documentation and hide details from high level callers.

Up to you if you like it or not...

Reply:
Would prefer to remove them because theres no real use for them as wrappers... `prepareForCheckpoint` isn't ever invoked, J9 invokes dispatcher reinit directly

Reply:
>We gain a bit on self-documentation and still hide details from high level callers.

`expandThreadPool` has been split up so that high level caller takes care of resetting thread count and then invokes reinit on dispatcher. Its more correct to do them independently, In some sense, high level caller need to know more details!

Reply:
Sorry ignore the previous messages, I misunderstood the initial suggestion.

Reply:
Thats a good idea, I will go ahead a make the change.